subprojects/signing/signing.gradle

```diff
@@ -23,6 +23,8 @@ dependencies {
     compile project(':plugins')
     compile project(':maven')
```

Comment:
Just curious, why do we have a compile dependency on maven here?

Reply:
Because of `SignatureExtension.signPom(MavenDeployment)` 😕